src/App.jsx

```diff
@@ -1,104 +1,31 @@
 import './App.scss';
+import { PostInfo } from './components/PostInfo/PostInfo';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+ import postsFromServer from './api/posts.json';
+ import commentsFromServer from './api/comments.json';
+ import usersFromServer from './api/users.json';
 
+ function getUserByPostId(postId) {
+  return usersFromServer.find(user => user.id === postId);
+}
+
+function getCommentByPostId(postId) {
+  return commentsFromServer.filter(comment => comment.postId === postId);
+}
+
+ export const posts = postsFromServer.map(post=> ({...post,
+  user: getUserByPostId(post.userId),
```

Comment:
The `getUserByPostId` function is used here, but it should be `getUserByUserId` to correctly fetch the user by `userId`. This is a critical issue that needs to be fixed to ensure the correct user is associated with each post.